src/extension/background-script/actions/accounts/remove.js

```diff
@@ -1,15 +0,0 @@
-import state from "../../state";
```

Comment:
This is used in https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Onboard/TestConnection/index.tsx#L23

Reply:
Arg, thanks. Will re-add and check.

Reply:
Right, re-added it. But this does the same thing, no?:  
https://github.com/getAlby/lightning-browser-extension/blob/refactor/652_account-types/src/extension/background-script/actions/accounts/delete.ts  

I assume we could just use `delete` instead of `remove` or am I missing something here?

Reply:
Alright, will adjust in another PR later on.